docs/apache-airflow-providers-databricks/operators/sql.rst

```diff
@@ -113,10 +113,53 @@ Configuring Databricks connection to be used with the Sensor.
     :start-after: [START howto_sensor_databricks_connection_setup]
     :end-before: [END howto_sensor_databricks_connection_setup]
 
-Poking the specific table for existence of data/partition:
+Poking the specific table with the SQL statement:
 
 .. exampleinclude:: /../../tests/system/providers/databricks/example_databricks_sensors.py
     :language: python
     :dedent: 4
     :start-after: [START howto_sensor_databricks_sql]
     :end-before: [END howto_sensor_databricks_sql]
+
+
+DatabricksPartitionSensor
```

Comment:
How familiar are people with the concept of sensors? Should we include a line or two that this can be used to wait until a partition is available?

Reply:
Added more documentation for Sensors in general and for the Partition Sensor as well.